tests/copy.bats

```diff
@@ -21,6 +21,12 @@ load helpers
   buildah copy $cid ${TESTDIR}/randomfile
   buildah copy $cid ${TESTDIR}/other-randomfile ${TESTDIR}/third-randomfile ${TESTDIR}/randomfile /etc
   buildah rm $cid
+
+  cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json alpine)
+  root=$(buildah mount $cid)
+  buildah config --workingdir / $cid
+  buildah copy $cid ${TESTDIR}/*randomfile /etc
```

Comment:
Please put double quotes around `${TESTDIR}/*randomfile` so that the shell doesn't do the globbing, and add a loop to verify that copies all of the matching files ended up in the right place with the right contents.